source/data-api/custom-endpoints.txt

```diff
@@ -343,55 +345,26 @@ Write an Endpoint Function
 Every custom endpoint is associated with a :ref:`function <functions>`
 that runs whenever the endpoint receives an incoming request. In the
 function, you can import libraries from npm, connect to a linked MongoDB
-Atlas cluster, and call other serverless functions.
+Atlas cluster, and call other serverless functions. 
 
-.. tip::
-   
-   You define and edit endpoint handler functions on the
-   :guilabel:`Functions` page of the App Services UI or in the :ref:`/functions
-   <appconfig-functions>` folder of an app configuration directory.
+To define and edit endpoint handler functions:
```

Comment:
Let's be clearer that these are all alternative ways to do the same thing. Right now it reads sort of like a procedure despite using an unordered list.

- You can define and edit endpoint handler functions from the UI, CLI, _or Admin API_
- Depending on your workflow, you can define functions... (bulleted list below this)

Reply:
Good point! I thought of a workaround in the most recent version - I emphasized that you can create a function in the same page while creating a custom endpoint, and I listed the alternative methods below. 

Do you think it feels excessive to describe each alternative? I'm on the fence. The alternative would be something like :

`To define a new function when creating an endpoint
 in the App Services UI, select the + New Function option in the Function section. Depending on your workflow, you can also define and edit endpoint handler functions from the UI, CLI, or Admin API. For more information, see Define a Function or App Services Admin API.`